src/python-common/ceph/rgw/rgwam_core.py

```diff
@@ -612,99 +669,159 @@ def realm_rm_zone_creds(self, realm_token_b64):
 
         return (0, success_message, '')
 
-    def zone_create(self, realm_token_b64, zonegroup_name=None, zone_name=None,
-                    endpoints=None, start_radosgw=True):
-        if not realm_token_b64:
-            print('missing realm access config')
-            return False
-
-        realm_token_b = base64.b64decode(realm_token_b64)
-        realm_token_s = realm_token_b.decode('utf-8')
-
-        realm_token = json.loads(realm_token_s)
+    def zone_modify(self, realm_name, zonegroup_name, zone_name, endpoints, realm_token_b64):
 
-        access_key = realm_token['access_key']
-        secret = realm_token['secret']
+        if not realm_token_b64:
+            raise RGWAMException('missing realm access config')
```

Comment:
is a realm token equivalent to an access config? Intuitively I wouldn't think so. Is this some rgw terminology I'm not aware of?